src/@utils/cookies.ts

```diff
@@ -6,14 +6,32 @@ export enum SAME_SITE_OPTIONS {
   NONE = 'none'
 }
 
-export const DEFAULT_COOKIE_OPTIONS: {
-  expires: number | Date
-  sameSite: SAME_SITE_OPTIONS
-} = {
+interface CookieOptions {
+  expires?: number | Date
+  sameSite?: SAME_SITE_OPTIONS
+}
+
+function addTwoMonths(date: Date): Date {
+  const newDate = new Date(date)
+  newDate.setMonth(newDate.getMonth() + 2)
+  return newDate
+}
+
+export const DEFAULT_COOKIE_OPTIONS: CookieOptions = {
   expires: 365,
   sameSite: SAME_SITE_OPTIONS.STRICT
 }
 
+export const SESSION_COOKIE_OPTIONS: CookieOptions = {
+  expires: undefined, // undefined means session cookie
+  sameSite: SAME_SITE_OPTIONS.STRICT
+}
+
+export const TWO_MONTHS_COOKIE_OPTIONS: CookieOptions = {
+  expires: addTwoMonths(new Date()),
```

Comment:
suggestion(non-blocking): it would be sufficient / enough imo to just set the `expires` value to e.g. `60` for the TWO_MONTHS option. Would be a bit easier to read and reduce code complexity.